### PR TITLE
Set default mediapipe base path to (main config path)\(graph_name)

### PR DESCRIPTION
**Pull request by michalkulakowski**

### CODE REVIEW

File: src/test/mediapipeflow_test.cpp [hunk 1]
```diff
@@ -1900,12 +1901,14 @@ TEST_F(MediapipeConfigChanges, ConfigWithEmptyBasePath) {
...
     std::string configFilePath = directoryPath + "/config.json";
-    std::string graphFilePath = directoryPath + "/graph.pbtxt";
+    std::string graphFilePath = directoryPath + "/mediapipeGraph/graph.pbtxt";
 
     const std::string inputName{"in\""};
     const std::string newInputName{"in2\""};
 
     createConfigFileWithContent(configFileContent, configFilePath);
+    std::string defaultGraphDirectoryPath = directoryPath + "/mediapipeGraph";
+    mkdir(defaultGraphDirectoryPath.c_str(), 0777);
```
Comment (atobiszei) [inline src/test/mediapipeflow_test.cpp:1911]: You could use more c++'ish approach with
```suggestion
    std::filesystem::create_directories(directoryPath)
```

File: src/test/mediapipeflow_test.cpp [hunk 2]
```diff
@@ -1900,12 +1901,14 @@ TEST_F(MediapipeConfigChanges, ConfigWithEmptyBasePath) {
     std::string graphPbtxtFileContent = pbtxtContent;
     std::string configFileContent = configFileWithEmptyBasePath;
     std::string configFilePath = directoryPath + "/config.json";
-    std::string graphFilePath = directoryPath + "/graph.pbtxt";
+    std::string graphFilePath = directoryPath + "/mediapipeGraph/graph.pbtxt";
```
Comment (atobiszei) [inline src/test/mediapipeflow_test.cpp:1904]:
```suggestion
    std::string graphName = "mediapipeGraph";
    std::string graphFilePath = directoryPath + "/" + graphName + "/" + MediapipeGraphConfig::DEFAULT_GRAPH_CONFIG_FILE_NAME;
```

File: src/test/mediapipeflow_test.cpp [hunk 3]
```diff
@@ -1900,12 +1901,14 @@ TEST_F(MediapipeConfigChanges, ConfigWithEmptyBasePath) {
...
 
     const std::string inputName{"in\""};
     const std::string newInputName{"in2\""};
 
     createConfigFileWithContent(configFileContent, configFilePath);
+    std::string defaultGraphDirectoryPath = directoryPath + "/mediapipeGraph";
```
Comment (atobiszei) [inline src/test/mediapipeflow_test.cpp:1910]:
```suggestion
    std::string defaultGraphDirectoryPath = directoryPath + "/" + graphName;
```

File: src/test/mediapipeflow_test.cpp (hunk 2 above)
Comment (atobiszei) [reply src/test/mediapipeflow_test.cpp:1904]: That would also require exposing static MediapipeGraphConfig class field. Right now it is just a locally scoped variable DEFAULT_GRAPH_FILENAME in mediapipegraphconfig.cpp.

**Review by atobiszei — APPROVED** [review]

**Review by rasapala — APPROVED** [review]
LGTM

**Pull request merged**